### PR TITLE
accommodations cannot be seen / edited

### LINKED ISSUE #303: Accommodations cannot be seen / edited

### PR DESCRIPTION
**Pull request by Guekka**
Fix #303

### CODE REVIEW

**Comment by Apoorva64** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit d5d9697efa26b0cafe128d3b9357266e6298f09d. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=640ee8fb56da4d6500e89717&branchName=308,303-accommodations-cannot-be-seen-edited)

---


<details>
<summary>✅ Successfully ran 2 targets </summary>

- [`nx serve backend-api`](https://cloud.nx.app/runs/JnkEV4W9KQ)
- [`nx affected --target=lint,test,build --parallel=3`](https://cloud.nx.app/runs/pgSXDo20ku)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: apps/backend/api/src/controllers/patient.accommodation.controller.ts [hunk 1]
```diff
@@ -55,28 +55,28 @@ export const addRelatedAccommodationHandler = async (
   try {
     const patient = await prisma.patient.findUnique({
       where: {
-        id: req.params.id,
+        id: req.body.id,
```
Comment (Apoorva64) [inline apps/backend/api/src/controllers/patient.accommodation.controller.ts:58]: No.

Comment (Guekka) [reply apps/backend/api/src/controllers/patient.accommodation.controller.ts:58]: It's a post request. Why wouldn't we use the body?

File: libs/front-end/shared/common/src/lib/services/patient/patient.service.ts [hunk 2]
```diff
@@ -85,7 +85,7 @@ export class PatientService {
   ): Observable<RequestWrapper<Accommodation>> {
     return this.http.post<RequestWrapper<Accommodation>>(
       this.PATIENT_URL + id + '/accommodation',
```
[the remark this answers is not in the document]
Comment (Guekka) [reply libs/front-end/shared/common/src/lib/services/patient/patient.service.ts:87]: Why use the URL for info when we can use the body?

File: apps/backend/api/src/controllers/patient.accommodation.controller.ts (hunk 1 above)
Comment (Apoorva64) [reply apps/backend/api/src/controllers/patient.accommodation.controller.ts:58]: because it's not the convention

Comment (Apoorva64) [reply apps/backend/api/src/controllers/patient.accommodation.controller.ts:58]: https://www.mscharhag.com/api-design/rest-many-to-many-relations

Comment (Apoorva64) [reply apps/backend/api/src/controllers/patient.accommodation.controller.ts:58]: the first solution was used everywhere

Comment (Guekka) [reply apps/backend/api/src/controllers/patient.accommodation.controller.ts:58]: Alright. Feel free to fix it, or I will do it tomorrow

**Review by Apoorva64 — APPROVED** [review]

**Pull request merged**